docs/freecam-readme.md

```diff
@@ -0,0 +1,117 @@
+
+### Pause/Unpause Feature
+
+- The game can be entirely paused/unpaused, which is useful for taking a screenshot of a particular moment in time:
+  - Press `O` to pause the game. Press `O` again to unpause the game.
+  - While the game is paused, you can still move the camera using FPS mode, adjust camera zoom etc.
+
+### Clip Distance
+
+- In some scenes, the camera will clip into objects very easily. By pressing `F7` you can toggle to a shorter clip distance. Please note that doing this may cause graphical artifacts with shadows. In addition, ADV mode deliberately uses clipping to avoid seeing the characters own head - with this mode enabled and the camera in the default ADV position, you'll see the inside of the character's head.
+
+### F6 mode / magenta box
+
+- You can press `F6` instead of `F8` to enter FPS mode. If you do this, a magenta box will spawn directly underneath the player (it's the standard Unity cube).
+- In somniums, the box can be used to lift and push the player around.
+- If the magenta box gets in the way of the camera, hold right click while moving the mouse and you can rotate the camera around the magenta box.
+
+#### Known bugs and wierd behaviors
+
+As this is a just a hack of the game and a side project for me, some "bugs" might not be worth fixing. I'll state the known bugs below, but might not ever fix them:
+
+- Cameras don't revert to their original position when you press F9 while in a Cinematic
+  - Workaround: None
+- The camera may starts rotating randomly in somniums
+  - Workaround: Disable the FPS camera, move a bit, then enable it again. You can also try entering and leaving the main menu.
+- When a new version of the game is released, probably everything will break until a new version of the mod is issued.
+  - Workaround: wait for a new release of the mod
+- A few scenes look like they're in-game, but they are actually videos. Since it's a video, you cannot move the camera.
+  - Workaround: None
+- Menus Disappear in FPS mode
+  - Press `F9` to disable FPS mode when attempting to navigate menus
+  - Press `F10` to toggle GUI hiding, incase you forgot you left it on
+- In somniunms, the "choice" buttons and some other GUI are not hidden while making the choice. Immediately after making the choice, they go away, so this is not really a problem.
+  - Workaround: None
+- The slow-motion modes don't work in some cutscenes
+  - Workaround: Repeatedly hit the pause/play button `O` to get to the part you want
+
+## Reporting Bugs
+
+Please report bugs by raising an issue on this github repository. If you don't want to create a Github account, you can PM me @drojf0 on twitter.
+
+## Developer's Instructions / Reproduction Instructions
+
+See the [Developer Readme](Developer_Readme.md) for reproduction instructions and various other useful information. You may be able to port the code to other Unity games.
```

Comment:
this is linking to a different filename which does not exist.

Reply:
Thanks, looks like slavonomics fixed it in ed0d34a8e2dabb597f2c71918de058daafad0c41